packages/RawButton/test/RawButton.spec.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import RawButton from "../src";
+
+const noop = () => {};
+
+const pressEnter = { key: "Enter" };
+const pressSpace = { key: " " };
+
+function renderComponent(props = {}) {
+  const defaultProps = {
+    onClick: noop,
+  };
+
+  return render(
+    <RawButton {...defaultProps} {...props}>
+      {props.children || "happy button"}
+    </RawButton>
+  );
+}
+
+describe("RawButton", () => {
+  it("should render with default props", () => {
```

Comment:
On the rails side we say `it('renders with default props', ...)` — I think I prefer that. Any opinions?

Reply:
Fine with me, pick one and stick with it 👍

Reply:
In general, I favour fewer words when possible, so dropping "should" in all test cases is fine with me.  Do we have consensus to make this a standard for the repo?